database/src/main/java/com/firebase/ui/database/FirebaseIndexRecyclerAdapter.java

```diff
@@ -59,8 +59,11 @@
  * </pre>
  *
  * @param <T>  The Java class that maps to the type of objects stored in the Firebase location.
- * @param <VH> The ViewHolder class that contains the Views in the layout that is shown for each object.
+ * @param <VH> The ViewHolder class that contains the Views in the layout that is shown for each
+ *             object.
+ * @deprecated use {@link com.firebase.ui.database.adapter.FirebaseIndexRecyclerAdapter} instead
  */
+@Deprecated
```

Comment:
I experimented with the idea above, you can see what would happen here:
https://github.com/samtstern/FirebaseUI-Android/commit/5d2d34b802c3295505dfd1144cad8f58abd79814

This makes the deprecation blow much softer, still not "ideal".  What do you think of something like that?